app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  before_save { self.email = email.downcase }
```

Comment:
Be sure to write a test for this functionality. Check that emails get downcased :)